src/modules/audio/input/input.js

```diff
@@ -0,0 +1,151 @@
+/*
+//  input.js
+//
+//  Created by Kalila L. on May 11th, 2021.
+//  Copyright 2021 Vircadia contributors.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+*/
+
+import Log from '../../debugging/log.js';
+
+export class AudioInput {
```

Comment:
Filename should match class name - e.g., AudioInput.js.

Also, `export` statement should be at end of file.